go/test/endtoend/vtgate/schema/schema_test.go

```diff
@@ -286,58 +281,6 @@ func checkTablesCount(t *testing.T, tablet *cluster.Vttablet, count int) {
 	assert.Equal(t, len(queryResult.Rows), count)
 }
 
-// testCopySchemaShards tests that schema from source is correctly applied to destination
-func testCopySchemaShards(t *testing.T, source string, shard int) {
```

Comment:
I removed the `CopySchemaShard` client command tests since this client command is no longer used anywhere that I could find so we can kill it off.